frontend/src/components/grantaccess/form/ForestClientInput.vue

```diff
@@ -32,15 +32,22 @@ const verifyForestClientNumber = async (forestClientNumbers: string) => {
     forestClientNumberVerifyErrors.value = [];
 
     // split by commas and spaces
-    let forestNumbers = forestClientNumbers.split(',').map(num => num.trim());
+    let forestNumbers = forestClientNumbers.split(',').map((num) => num.trim());
 
-    for (const item of forestNumbers) {
+    for (const forestClientNumber of forestNumbers) {
+        if (!isForestClientNumberAdded(forestClientNumber)) {
+            forestClientNumberVerifyErrors.value.push(
+                `Client ID ${forestClientNumber} has already been added.`
+            );
```

Comment:
Jota, not just rename the method, got to be careful.
Did you check again for "if" that makes sense to you for what's being executed below?

Reply:
sorry about that, at the time i did not thought it would be hard to understand.